src/models/rest.ts

```diff
@@ -319,9 +319,7 @@ export class RESTManager {
         }
       }
       const form = new FormData()
-      for (const file of files) {
-        form.append(file.name, file.blob, file.name)
-      }
+      files.map((file, index) => form.append(`file${index + 1}`, file.blob, file.name))
```

Comment:
Don't use `map`, use `forEach`

Reply:
.forEach() is typically used when the data is to be mutated.

Reply:
It's the other way, a map is used when data it to be mutated since it returns the result of the mutation.

Reply:
Map allocates new Array, forEach is better in this case.

Reply:
> It's the other way, a map is used when data it to be mutated since it returns the result of the mutation.
@ZiomaleQ I think you are mistaken .map() creates a new array and and returns that array.